app/src/lib/markdown-filters/video-url-regex.ts

```diff
@@ -0,0 +1,14 @@
+import { escapeRegExp } from 'lodash'
```

Comment:
Can we use the `escapeRegExp` function from `regex.ts` instead? I'd rather we made a conscious decision to move over to lodash's method across the code base than to mix

```suggestion
import { escapeRegExp } from '../helpers/regex'
```